documentation/concepts/benefits_of_flex.rst

```diff
@@ -0,0 +1,116 @@
+the ordered volume relates only to the selected time window, while the curtailed volume may include volumes outside of the selected time window.
+For example, an asset that runs an all-or-nothing consumption process of 2 hours can be ordered to curtail consumption for 1 hour, but will in effect stop the entire process.
+In this case, the curtailed volume will be higher than the ordered volume, and the platform will take into account the total expected curtailment in its calculations.
+
+Shifting
+^^^^^^^^^^^^^^
+
+Shifting happens when an asset delays or advances its energy production or consumption.
+A defining feature of shifting is that total production or consumption at the end of the flexibility opportunity remains the same.
+
+- An example of delaying consumption is when a charging station postpones the charging process of an electric vehicle.
+- An example of advancing consumption is when a cooling unit starts to cool before the upper temperature bound was reached (pre-cooling).
+
+Shifting offers may specify some freedom in terms of how much energy can be shifted.
+In these cases, the user can select the energy volume (in MWh) to be ordered, within constraints set by the relevant Prosumer.
+This energy volume represents how much energy is shifting into or out of the selected time window.
+The net effect of a shifting action is measured in terms of an energy-time volume (see the flexibility metrics in the :ref:`portfolio` page).
+This volume is a multiplication of the energy volume being shifted and the duration of that shift.
+
+
+.. _activation_profits
+
+Profits of flexiblity activation 
+---------------
+
+The realised value from activating flexibility opportunities has to be computed and accounted for.
+Both of these activities depend on the context in which FlexMeasures is being used, and we expect that it will be often have to implemented in a custom manner (much as the actual scheduling optimization).
+
+.. note:: Making it possible to configure custom scheduling and value accounting is on the roadmap for FlexMeasures.
+
+Computing value
+^^^^^^^^^^^^^^^^
+
+The computation of the value is what drives the scheduling optimization. This value is usually monetary, and in that case there should be some form of market configured. This can be a constant or time-of-use tariff, or a real market. However, there are other possibilities, for instance if the optimisation goal is to minimise the CO2 consumption. The realised value is avoided CO2, whcih is probably not easy to translate into a monetary value.
+
+
+Accounting / Sharing value
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The realisation of payments is outside of FlexMeasures scope, but it can provide the accounting to enable them (as was said above, this is usually a part of the optimisation problem formulation).
+
+However, next to fueling algorithmic optimization, the value of energy flexibility also drives project partipation. Accounting plays an important role here.
```

Comment:
What do you mean with "the value of energy flexibility drives project participation"?